api/src/main/java/org/pmiops/workbench/mail/MailServiceImpl.java

```diff
@@ -59,8 +58,8 @@
 @Service
 public class MailServiceImpl implements MailService {
 
-  public static final ImmutableMap<EgressRemediationAction, String> EGRESS_REMEDIATION_ACTION_MAP =
-      ImmutableMap.of(
+  public static final Map<EgressRemediationAction, String> EGRESS_REMEDIATION_ACTION_MAP =
```

Comment:
a variety of misc updates like this.  `Map.of()` and `List.of()` are newer and native to Java, rather than requiring a library